pkg/config/config.go

```diff
@@ -297,19 +297,22 @@ type IngressConfig struct {
 type SIPConfig struct {
 }
 
-// not exposed to YAML
 type APIConfig struct {
 	// amount of time to wait for API to execute, default 2s
-	ExecutionTimeout time.Duration
+	ExecutionTimeout time.Duration `yaml:"execution_timeout,omitempty"`
 
-	// amount of time to wait before checking for operation complete
-	CheckInterval time.Duration
+	// min amount of time to wait before checking for operation complete
+	CheckInterval time.Duration `yaml:"check_interval,omitempty"`
+
+	// max amount of time to wait before checking for operation complete
+	MaxCheckInterval time.Duration `yaml:"max_check_interval,omitempty"`
```

Comment:
I guess it makes no sense for this value to be higher than ExecutionTimeout right? you are just making it configurable.

Reply:
i'll make it configurable in cloud but i assumed it was omitted in oss for a good reason. in oss the timing will be 100, 200, 300x6 then it will time out. we'll set a lower retry rate in cloud. we'll also stop retrying now if the user goes away

Reply:
oh i guess it has to be configurable here...